### ISSUE TITLE
TF2 + GH

### ISSUE THREAD

**Issue by Rababalkhalifa** [tensorflow/tensorflow#47908]
![image](https://user-images.githubusercontent.com/24195074/111758231-24de0700-8894-11eb-9778-1716c8fdb9dd.png)

I am getting list of error as I am trying to change TF2 BERT from using tfhub bert to hugging face bert.

The accuracy and results are not similar to using tfhub bert besides I am getting all the error above.

I am using this:
from transformers import BertConfig, AutoTokenizer, TFAutoModel
model_name = "bert-base-uncased"
tokenizer = AutoTokenizer.from_pretrained(model_name)
config = BertConfig.from_pretrained(model_name, num_labels=2, return_dict=True,output_hidden_states=True,use_cache=True,output_attentions=True)
bert = TFAutoModel.from_pretrained(model_name, config=config)

**Comment by amahendrakar** [comment]
@Rababalkhalifa,
Could you please let us know the TensorFlow version you are using? 

I did not face any warnings while running the code with TF v2.4.1. Please find the gist of it [here](https://colab.research.google.com/gist/amahendrakar/dd598bcdb663e7373ce48b960beb90f4/47908.ipynb). Thanks!

**Comment by amahendrakar** [comment]
Also, you can suppress the warning messages by changing the log level at the start of the program 
```
import os
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2' 
import tensorflow as tf
```

or you can set the [autograph verbosity](https://www.tensorflow.org/api_docs/python/tf/autograph/set_verbosity
) level using

```
import os
import tensorflow as tf
os.environ['AUTOGRAPH_VERBOSITY'] = 1
```
Thanks!

**Comment by Rababalkhalifa** [comment]
I am using tf 2.4.1 in colab.
The code was fine before changing to HF layers. All what I did is changing the tokenizer and bert parts.
using batch encoder and replacing this:
  encoder = hub.KerasLayer(tfhub_handle_encoder, trainable=True, name='encoder')
with:
  net = bert(inputs['input_word_ids'], attention_mask=inputs['input_mask'])[0]

**Comment by amahendrakar** [comment]
> I am using tf 2.4.1 in colab.

@Rababalkhalifa,
Could you please share the Colab notebook with us, so that we can look into this? Thanks!

**Comment by Rababalkhalifa** [comment]
contact me in r.alkhalifah @ hotmail

**Comment by amahendrakar** [comment]
@ymodak,
I was able to reproduce the issue with TF v2.4.1, TF v2.5.0-rc0 and TF-nightly. Please find the gist of it [here](https://colab.research.google.com/gist/amahendrakar/734fce9c108fedd0ec88613986a9750e/47908.ipynb). Thanks!

**Comment by ymodak** [comment]
I think [huggingface/transformers ](https://github.com/huggingface/transformers)repository is right place for this issue since the you have imports from transformers module. Please try posting this issue on transformers repository. Thanks!
> from transformers import BertConfig, AutoTokenizer, TFAutoModel
